### DISCUSSION TITLE
Overtime/Undertime display

### DISCUSSION THREAD

**Discussion by dhoernel** [kimai/kimai#4345] [Questions & Answers]
We have problems to understand how and where the overtimes/undertimes in the green box next to "Check" are calculated:

![image](https://github.com/kimai/kimai/assets/48748967/b161d809-9b1a-4b9e-9910-f2c9aa859c6c)

Can someone give me a hint where I can find more information / what is the related source code file?

**Comment by kevinpapst** [comment]
This is not Kimai, this is the Controlling Plugin by MR Software.

Unfortunately you won't get any support for that here or anywhere for that matter 😢 

https://www.kimai.org/blog/2023/controlling-plugin-mr-software/

**Reply by dhoernel** [reply]
Ok, I understand. However, it would be nice to have this overtimes/undertimes display as a Kimai "main" feature. This is what employees will probably look at in the first place to see whether they have overtimes/undertimes. Or is there another place in Kimai to spot this information easily?

**Reply by kevinpapst** [reply]
Yeah, who needs to earn money... would be nice if the software would be for free and cover 100% of your use-cases.

**Reply by dhoernel** [reply]
Sorry, but this answer was not very helpful...

**Reply by kevinpapst** [reply]
Oh right, sorry for giving you a free software and free support.
